Sources/ApolloCodegenLib/Templates/SelectionSetTemplate.swift

```diff
@@ -29,20 +30,30 @@ struct SelectionSetTemplate {
   func render(typeCase: IR.SelectionSet) -> String {
     TemplateString(
     """
+    \(SelectionSetNameDocumentation(typeCase))
     public struct As\(typeCase.renderedTypeName): \(schema.name).TypeCase {
       \(BodyTemplate(typeCase))
     }
     """
     ).description
   }
 
+  func SelectionSetNameDocumentation(_ selectionSet: IR.SelectionSet) -> TemplateString {
+    """
+    /// \(generatedSelectionSetName(
```

Comment:
Do we want to add `MARK: ` here so the names show up in the jump bar in xcode?

Reply:
Sure. Will do